extensions/2.0/Khronos/KHR_texture_float_bc6h/README.md

```diff
@@ -0,0 +1,219 @@
+﻿# KHR_texture_float_bc6h
+
+## Contributors
+
+* Rickard Sahlin, mailto:rickard.sahlin@ikea.com  
+* Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)  
+* Don McCurdy, Google, [@donrmccurdy](https://twitter.com/donrmccurdy)  
+* Norbert Nopper, mailto:nopper@ux3d.io  
+
+Copyright (C) 2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.  
+This extension requires the KHR_texture_basisu extension to be supported.  
+
+## Overview
+
+This extension adds the ability to specify textures using KTX v2 images with float format as defined by VK_FORMAT_BC6H_UFLOAT_BLOCK.
+
+When the extension is used, it's allowed to use value `image/ktx2` for the `mimeType` property of images that are referenced by the `source` property of `KHR_texture_float_bc6h` texture extension object.
+
+At runtime, engines are expected to check hardware support for VK_FORMAT_BC6H_UFLOAT_BLOCK, if not present the texture should be transcoded converted to a suitable supported format.
+
+## glTF Schema Updates
+
+The `KHR_texture_float_bc6h` extension is added to the `textures` object and specifies a `source` property that points to the index of the `image` which defines a reference to the KTX v2 image in VK_FORMAT_BC6H_UFLOAT_BLOCK.
+
+The following glTF will load `image.ktx2` in clients that support this extension, and otherwise fall back to `image.png`.
```

Comment:
There is no fallback from a floating-point format to a regular PNG unless the range is restricted to `0.0 ... 1.0`.

Reply:
So you are saying that we do not want a possible LDR fallback?

Reply:
That entirely depends on the goals and usage. For example:
- No real fallback for non-prefiltered IBLs.
- Questionable (to say the least) fallback for WCG.
- Limited fallback for standard-range high-precision content.

Reply:
Removed fallback